pkg/urbit/include/vere/vere.h

```diff
@@ -145,16 +145,17 @@
         } siz;
 
         struct {
-          c3_y* lin_y;                      //  current line (utf8)
-          c3_w  byt_w;                      //  utf8 line-length
-          c3_w  wor_w;                      //  utf32 line-length
-          c3_w  sap_w;                      //  escape chars in line
-          c3_w  cus_w;                      //  cursor position
+          u3_noun lin;                      //  bottom line (stub)
```

Comment:
I believe we've discussed this before, but I've forgotten: why is the bottom line stored in the driver state as a noun instead of a processed `uv_buf_t` or whatever?

Reply:
Because a blit might _partially_ overwrite the line. Using a `uv_buf_t` that would of course still be trivial for the plaintext case, but keeping _styling_ correct when that's in play would require parsing the `uv_buf_t` for escape sequences etc etc.

Reply:
I see, I hadn't realized that `mir.lin` isn't just the last blit, but has been processed with `+wail:klr:format` and `+pact:klr:format`.

Reply:
On second thought, I'm slightly less convinced. `+wail` as called is basically for line-wrapping, but it assumes that each `@c` is an individual glyph. This assumption is usually correct, but not always (see "grapheme clusters" ...). I'm not sure how much of a problem this is, but I wonder if we should bite the bullet and pin the spinner to the bottom right corner, or even a temporary new line.

Reply:
> or even a temporary new line

(Not sure that would work in the first place, but) honestly if we're going there we should just add a persistent status bar... and write a handful of `//TODO`s for things to put in it.

Reply:
Having thought about this some more, I'm comfortable with the status quo. The spinner as designed simply conflicts with a 2-dimensional output array, and that will have to be solved at some point. Since it's purely a feature of the runtime, it can be changed without impact to the %dill protocol.